guestbook/redis-follower/run.sh

```diff
@@ -14,12 +15,11 @@
 # limitations under the License.
 
 # [START gke_redis_follower_run]
-#!/bin/bash
 
 if [[ ${GET_HOSTS_FROM:-dns} == "env" ]]; then
   redis-server --replicaof ${REDIS_LEADER_SERVICE_HOST} 6379
 else
   redis-server --replicaof redis-leader 6379
 fi
 
-# [END gke_redis_follower_run]
\ No newline at end of file
+# [END gke_redis_follower_run]
```

Comment:
This new line at the end of the file is technically unrelated to the problem (and out of scope), so let me know if I should undo it.